src/components/SecondaryPanes/Frames/Frame.js

```diff
@@ -148,3 +156,4 @@ export default class FrameComponent extends Component<FrameComponentProps> {
 }
 
 FrameComponent.displayName = "Frame";
+FrameComponent.contextTypes = contextTypes;
```

Comment:
I'd prefer not drying up the `contextTypes`

1. it is clearer to read the types
2. it is possible we'll have different context types for each component, i.e. does this use shortcuts?